syntax = "proto3";

package fbsrankings.messages.query;

message LatestSeasonWeekValue {
    string season_id = 1;
    int32 year = 2;
    optional int32 week = 3;
}

message LatestSeasonWeekResult {
    optional LatestSeasonWeekValue latest = 1;
}

message LatestSeasonWeekQuery {
    string query_id = 1;
}
